_activities/cwp.md

```diff
@@ -1,181 +1,58 @@
 ---
-title: "Community White Paper"
-author: "Peter Elmer"
+title: "A Roadmap for HEP Software and Computing R&D for the 2020s"
```

Comment:
I agree with the page title change but it would be good to keep Community White Paper as the menu entry.  I thought I added the support for a `menu` attribute but it doesn't seem the case. Easy to implement in `_includes/navbar.ext`. I'll try to open a PR later if you don't do it yourself.

Reply:
In the first rebase I went back to the old name because the text in the navbar became too long. If you want to change it afterwards that's fine for me.